### DISCUSSION TITLE
How do I connect to the Induction Matrix?

### DISCUSSION THREAD

**Discussion by IBuildVans** [MikaylaFischler/cc-mek-scada#269] [Support] [answered]
Hello!   Loving the program so far.   I setup a test reactor/turbine and would like to add the Induction Matrix.   Ive tried connecting to the RTU/Supervisor/Coordinator and I cannot get it to show as connected.   I am using a wired modem connected to an Induction Matrix port.   Anyone have any suggestions?

**Comment by MikaylaFischler** [comment, marked as answer]
So that would need a wired modem connecting the induction matrix to an RTU, then the RTU config (`rtu/config.lua` on the computer) would need a definition like this in the `RTU_DEVICES` table (example for if the device connects to the network as `inductionPort_0`:

```lua
    {
        name = "inductionPort_0",
        index = 1,
        for_reactor = 0
    }
```

**Reply by IBuildVans** [reply]
Thank you very much for the answer!   I was close.   I had everything connected. I just still had a 1 when it should have been  "for_reactor = 0".   I had to wait unntil I was home from work to try it out.  Thank you again.

**Reply by MikaylaFischler** [reply]
Happy to help!

**Reply by Overhandmite** [reply]
So I tried what you posted here and it crashes my rtu as it says the line was expected to end? I’ve gotten everything else hooked up fine but the induction matrix

**Reply by PepijnTrampe** [reply]
make sure that you have a comma at one line like
`    {
        name = "boilerValve_0",
        index = 1,
        for_reactor = 1
    }, 
    {
        name = "turbineValve_0",
        index = 1,
        for_reactor = 1
    },  
    {
       name = "inductionPort_0",
       index = 1,
       for_reactor = 0
    } 
}`
if you have 3 divices the first 2 have a comma to close the loop and the last one not atherwise you got a line error
